src/test/java/unrc/dose/UserTest.java

```diff
@@ -0,0 +1,59 @@
+package unrc.dose;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import org.junit.BeforeClass;
+import org.junit.AfterClass;
+
+import org.javalite.activejdbc.Base;
+import org.javalite.activejdbc.DB;
+import spark.Spark;
+
+import unrc.dose.User;
+import org.junit.Test;
+
+
+public class UserTest {
+	
+	@BeforeClass
+  	public static void beforeAll() {
+    	App.main(null);
+
+		Spark.awaitInitialization();
+      	Base.open();
+  	}
+
+  	@AfterClass
+  	public static void tearDown() {
+    	Spark.stop();
+    	Base.close();
+  	}
+
+  	/**
+	 * test that verifies that the password has been modified successfully
+	 * @result the email is not registered in the DB
+	*/
+	@Test
+	public void resetPasswordNotSuccessfully(){
```

Comment:
The name should be `updatePasswordUnsuccessful`, but also you should consider that there are several possible reasons for a password update to fail.
 - User doesn't exist (no user with provided email)
 - The provided password is the same as the current password
 - The provided password is empty or doesn't comply with password policies (e.g.: minimum length)